plugins/chat/assets/javascripts/discourse/templates/components/chat-channel-settings-view.hbs

```diff
@@ -42,17 +41,19 @@
       </div>
     </div>
   {{/unless}}
+  <div class="chat-retention-info">{{d-icon "info-circle"}}{{i18n "chat.settings.retention_info" days=this.lengthOfRetention}}</div>
```

Comment:
Oh you can directly use the site setting here:

```
<div class="chat-retention-info">{{d-icon "info-circle"}}{{i18n "chat.settings.retention_info" days=this.siteSettings.chat_channel_retention_days}}</div>
```

So you won't need that `lengthOfRetention` variable in the JS either

Reply:
oh cool, thanks